client/src/components/Header/index.js

```diff
@@ -13,11 +13,11 @@ const Header = () => {
       <div className="container flex-column justify-space-between-lg justify-center align-center text-center">
         <Link className="text-dark" to="/">
           <h1 className="m-0" style={{ fontSize: '3rem' }}>
```

Comment:
I see this was brought in from the starter code, but we should try to avoid inline styling whenever possible.